src/hasSubObject.test.ts

```diff
@@ -92,25 +92,49 @@ describe("typing", () => {
       hasSubObject({ a: { b: 4, c: "c" } }, { a: { b: 1, c: 2 } });
     });
 
-    test("narrows type", () => {
+    it("narrows with empty object", () => {
```

Comment:
this is actually "**doesn't** narrow on empty object"

Reply:
@eranhirsch I thought about this but then thought that this test is about narrowing when using a type guard function and that it works properly in case of using an empty object: "it properly narrows type when using an empty object for the `subObject`". So I'm not sure we should use `doesn't` here.

In the test, we ensure that the type narrowing properly works when using an empty object in both branches of the condition.

Do you think I still should use `doesn't` here?